index.js

```diff
@@ -53,7 +54,11 @@ var initConfig = function(projectConfig) {
   util._extend(config, require('./config.js'));
   util._extend(config, projectConfig);
   if (process.env.hasOwnProperty('GCLOUD_TRACE_CONFIG')) {
-    util._extend(config, require(process.env.GCLOUD_TRACE_CONFIG));
+    var config_path = process.env.GCLOUD_TRACE_CONFIG;
+    if (config_path[0] !== path.sep) {
```

Comment:
Not a biggie, this should work (except for windows drive letter prefixes?), but I prefer to put all the responsibility on someone else, with something like [`path.resolve(process.env.GCLOUD_TRACE_CONFIG)`](https://nodejs.org/dist/latest-v4.x/docs/api/path.html#path_path_resolve_from_to)